src/frontend/apps/standalone_site/src/features/Playlist/components/PlaylistPage.tsx

```diff
@@ -0,0 +1,159 @@
+              <Button primary a11yTitle={intl.formatMessage(messages.create)}>
+                {intl.formatMessage(messages.create)}
+              </Button>
+            </Box>
+          )}
+        </Box>
+        {isLoading && <Spinner />}
+        {((!isLoading && !data && !isError) || isError) && (
+          <Box
+            background="content-background"
+            margin="auto"
+            pad="medium"
+            round="small"
+          >
+            <Text size="large">{intl.formatMessage(messages.error)}</Text>
+            <Box margin={{ horizontal: 'auto', top: 'medium' }}>
+              <Button
+                a11yTitle={intl.formatMessage(messages.retry)}
+                onClick={() => {
+                  refetch();
+                }}
+                primary
+              >
+                {intl.formatMessage(messages.retry)}
+              </Button>
+            </Box>
+          </Box>
+        )}
+        {!isLoading &&
+          !isError &&
+          data &&
+          (data.results.length === 0 ? (
+            <Box
+              background="content-background"
+              margin="auto"
+              pad="medium"
+              round="small"
+            >
+              <Text size="large">
+                {intl.formatMessage(messages.noPlaylists)}
+              </Text>
+              <Box margin={{ horizontal: 'auto', top: 'medium' }}>
+                <Button title={intl.formatMessage(messages.create)} primary />
+              </Box>
+            </Box>
+          ) : (
+            <SortableTable
+              loading={isLoading}
+              title={
+                <Box direction="row">
+                  <CheckListIcon width={30} height={30} />
+                  <Text margin={{ left: 'small' }}>
+                    {intl.formatMessage(messages.tableTitle, {
+                      item_count: data.count,
+                    })}
+                  </Text>
+                </Box>
+              }
+              items={data.results}
+              selectable
+              onSelectionChange={(items) => {
+                console.log('new selection', items);
+              }}
+              sortable
+              sortBy={sorts}
+              currentSort={currentSort}
+              onSortChange={(newSort) => {
+                setCurrentSort(newSort);
+                return data.results;
+              }}
+              paginable
+              numberOfItems={data.count}
+              pageSize={pageSize}
+              onPageChange={(newPage) => {
+                setCurrentPage(newPage);
+                return data.results;
+              }}
+            >
+              {(item) => (
+                <Box flex direction="row" align="center">
+                  <Box basis="30%">{item.title}</Box>
+                  <Box basis="50%">{item.lti_id}</Box>
+                  <Box basis="20%">{item.consumer_site}</Box>
```

Comment:
I'm not sure about the item displayed (may be there was a spec somewhere?) but I guess the `consumer_site` here will be a UUID, I guess it is not that helpful.
So maybe not for this PR, but we may improve the API to give more details about the consumer site.

Reply:
Your guess is right as discussed in daily few days ago.
Currently we mainly get uuid and the api should be pumped up (and I'm not even sure we want to display the consumer_site here tbh) but yeah it's probably for an other PR